pkg/k8shandler/collection.go

```diff
@@ -92,7 +92,7 @@ func CreateOrUpdateCollection(cluster *logging.ClusterLogging) (err error) {
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
 				if !reflect.DeepEqual(rsyslogStatus, cluster.Status.Collection.LogCollection.RsyslogStatus) {
-					logrus.Infof("Updating status of Rsyslog")
+					logrus.Info("Updating status of Rsyslog")
```

Comment:
I think it is useful to print this, but not more than once if looping in the `RetryOnConflict`